### DISCUSSION TITLE
Can I download packages from github straight to all my user devices or are there request limits?

### DISCUSSION THREAD

**Discussion by otisbison** [community/community#41016] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can I use public GitHub package release URL as source to deploy packages to my users or is there usage limits on that?

I've seen a few different answers (yes, no) to that question. Closest thing I found was possibly this page: https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases

**Comment by ossanna16** [comment]
Hi there @otisbison and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
